src/client/utils/dom.js

```diff
@@ -339,9 +339,13 @@ export function isIframeWithoutSrc (iframe) {
     const iframeSrcLocation      = iframeLocation.srcLocation;
     const iframeDocumentLocation = iframeLocation.documentLocation;
 
-    if (iframeDocumentLocation === null) // is a cross-domain iframe
+    // NOTE: is a cross-domain iframe
+    if (iframeDocumentLocation === null)
         return false;
 
+    if (iframe.contentWindow[INTERNAL_PROPS.documentWasCleaned])
```

Comment:
Add comment
// NOTE:  after 'document.write ' call for iframe with/without  src we will process it as iframe without src